src/scripts/multisig.py

```diff
@@ -1,39 +1,84 @@
-"""
-deployment script for creating a Tellor multisig on Algorand
-"""
+import json
+from algosdk.v2client.algod import AlgodClient
+from algosdk import account, encoding, mnemonic
+import base64
 import os
-import sys
+from algosdk.future.transaction import *
 
-from algosdk.v2client.algod import AlgodClient
-from dotenv import load_dotenv
+# Change these values with mnemonics
+mnemonic1 = os.getenv("MNEMONIC1")
+mnemonic2 = os.getenv("MNEMONIC2")
+mnemonic3 = os.getenv("MNEMONIC3")
+mnemonic4 = os.getenv("MNEMONIC4")
+# never use mnemonics in production code, replace for demo purposes only
+
+# For ease of reference, add account public and private keys to
+# an accounts dict.
+
+private_key_1 = mnemonic.to_private_key(mnemonic1)
+account_1 = mnemonic.to_public_key(mnemonic1)
+
+private_key_2 = mnemonic.to_private_key(mnemonic2)
+account_2 = mnemonic.to_public_key(mnemonic2)
+
+private_key_3 = mnemonic.to_private_key(mnemonic3)
+account_3 = mnemonic.to_public_key(mnemonic3)
+
+private_key_4 = mnemonic.to_private_key(mnemonic4)
+account_4 = mnemonic.to_public_key(mnemonic4)
```

Comment:
try storing the accounts in the `Accounts` object in the `utils`. It should help with organizing the keys.